README.rst

```diff
@@ -28,114 +29,79 @@ README for Pylint - https://pylint.pycqa.org/
    :width: 200
    :alt: Tidelift
 
-.. list-table::
-   :widths: 10 100
+What is pylint ?
+================
 
-   * - |tideliftlogo|
-     - Professional support for pylint is available as part of the `Tidelift
-       Subscription`_.  Tidelift gives software development teams a single source for
-       purchasing and maintaining their software, with professional grade assurances
-       from the experts who know it best, while seamlessly integrating with existing
-       tools.
+.. Do not modify this without also modifying doc/index.rst
 
-.. _Tidelift Subscription: https://tidelift.com/subscription/pkg/pypi-pylint?utm_source=pypi-pylint&utm_medium=referral&utm_campaign=readme
+Pylint is a `static code analyser`_ for Python 2 or 3. The latest version supports Python
+3.7.2 and above.
 
+.. _`static code analyser`: https://en.wikipedia.org/wiki/Static_code_analysis
 
-======
-Pylint
-======
+Pylint analyses your code without actually running it. It checks for errors, enforces a coding
+standard, looks for `code smells`_, and can make suggestions about how the code could be refactored.
 
-**It's not just a linter that annoys you!**
```

Comment:
Are we sure we want to remove the tagline?

Reply:
The tagline is still in the repo description, maybe that's enough or maybe we should change it to something descriptive, "A very configurable and thorough python static code analyzer`` for example ? I'm not a marketing person but I don't like the fact that we use "annoy" in our disclaimer 😄 Never saw a Coca cola add saying "Not just a beverage that makes you fat". How about "Code as you are" ?

Reply:
Don't think we should change it:
https://www.google.com/search?client=safari&rls=en&q=It%27s+not+just+a+linter+that+annoys+you&ie=UTF-8&oe=UTF-8

There seem to be some articles which use our tagline as their title. I don't mind having it at the top somewhere, but removing it is also fine.

Reply:
> Code as you are, not as you were.

🎸

Reply:
I saw that the title ``uncurled`` came from a [twitter thread](https://github.com/bagder/uncurled/issues/3#issuecomment-1091920617). maybe we can try to crowdsource the tagline 😄

Reply:
It would make sense to change it after #3512

Reply:
Agreed!